nmdc_server/app.py

```diff
@@ -57,8 +46,9 @@ async def lifespan(app: FastAPI):
         docs_url="/api/docs",
         openapi_url="/api/openapi.json",
         debug=settings.debug,
-        lifespan=lifespan,
     )
+    if static_path.exists() and static_path.is_dir():
```

Comment:
Since the app needs the static files to function, should this be a requirement? e.g.

```py
if not static_path.is_dir():
    raise Exception("Static files not found")
```